sandbox/concurrent_bert_etl/app_all_lambdas/jobs.py

```diff
@@ -0,0 +1,212 @@
+def bert_tess_fullframe_worker_1():
+    """Extract light curve data from one TESS full frame image.
+
+    This function is meant to be called from
+    :func:`bert_tess_fullframe_main` concurrently.
+
+    """
+    import os
+
+    import boto3
+    import numpy as np
+    from astropy.io import fits
+
+    np.seterr(all='ignore')
+
+    # Function to calculate circular mask.
+    # https://stackoverflow.com/questions/49330080/numpy-2d-array-selecting-indices-in-a-circle
+    def circular_mask(arr_shape, cx, cy, r):
+        y = np.arange(arr_shape[0])
+        x = np.arange(arr_shape[1])
+        return ((x[np.newaxis, :] - cx) ** 2 + (y[:, np.newaxis] - cy) ** 2) < (r ** 2)  # noqa
+
+    s3 = boto3.resource('s3')
+    bucket = s3.Bucket(name=os.environ.get('AWSBUCKETNAME'))
+    homedir = os.environ.get('HOME')
+
+    work_queue, done_queue, ologger = utils.comm_binders(
+        bert_tess_fullframe_worker_1)
+
+    # TODO: How to grab this from Lambda event for unit testing?
+    #
+    # Example event:
+    # {
+    #   "key": "tess/public/ffi/s0001/2018/206/1-1/tess2018206192942-s0001-1-1-0120-s_ffic.fits",  # noqa
+    #   "xpos": 1170,
+    #   "ypos": 1052,
+    #   "radius": 8,
+    #   "bright_pixel_threshold": 2.5
+    # }
+    #
+    # work_queue populated by bert_tess_fullframe_main()
+    for event in work_queue:
+        key = event['key']
+
+        # For signal
+        xpos = int(event['xpos'])
+        ypos = int(event['ypos'])
+        radius = int(event['radius'])
+
+        # For background
+        threshold = float(event['bright_pixel_threshold'])
+
+        filename = os.path.join(homedir, key.split('/')[-1])
+
+        # TODO: Find a way to not do this when testing locally to save money.
+        ologger.info(f'Downloading File[{filename}]')
+        bucket.download_file(
+            key, filename, ExtraArgs={"RequestPayer": "requester"})
+
+        with fits.open(filename) as pf:
+            # EXT 0: Primary header
+            # Observation time in BJD.
+            obstime = 0.5 * (pf[0].header['TSTOP'] + pf[0].header['TSTART'])
+
+            # EXT 1: cal
+            # Signal is inside a circle.
+            mask = circular_mask(pf[1].data.shape, xpos, ypos, radius)
+            signal = np.nanmedian(pf[1].data[mask])
+            # Background is everything else.
+            bg_arr = pf[1].data[~mask]
+            background = np.nanmedian(bg_arr[(bg_arr * threshold) < signal])
+
+        # Clean up
+        os.remove(filename)
+
+        # TODO: Do we need astropy.utils.misc.JsonCustomEncoder here?
+        # TODO: Do we need to pass more info for debugging?
+        # Pass data into the next AWS Lambda function.
+        done_queue.put({'frame': os.path.basename(filename),
+                        'midtime': obstime,
+                        'signal': signal,
+                        'background': background})
+
+
+# TODO: Do we need to say BOTTLE here?
```

Comment:
@jbcurtin , didn't you say I would need to specify `BOTTLE` in the binding here?